diff --git a/sdk/cargo-build-bpf/Cargo.toml b/sdk/cargo-build-bpf/Cargo.toml
index 888b59a101..86968aef4f 100644
--- a/sdk/cargo-build-bpf/Cargo.toml
+++ b/sdk/cargo-build-bpf/Cargo.toml
@@ -14,7 +14,6 @@ bzip2 = "0.4.3"
 cargo_metadata = "0.14.2"
 clap = { version = "3.1.5", features = ["cargo", "env"] }
 regex = "1.5.4"
-solana-download-utils = { path = "../../download-utils", version = "=1.10.11" }
 solana-sdk = { path = "..", version = "=1.10.11" }
 tar = "0.4.38"
 
